rpc/doc.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
```

Comment:
The style for doc.go files is to always start with "Package x is/defines...". See, for example, https://github.com/juju/juju/blob/main/domain/model/doc.go

Reply:
noted with thanks!